### DISCUSSION TITLE
Worker node is running on docker but shows inactive on the trino webUI

### DISCUSSION THREAD

**Discussion by CDhote** [trinodb/trino#24051] [Q&A]
I am new to Trino and evaluating Trino to query different data source. 
 I have deployed Trino using docker on two separate aws linux server, one server for the coordinator and another for worker. The trino set up is running on both the servers but don't know why worker is showing inactive. checked the logs but can't find any error. Please help.
![Trino web1](https://github.com/user-attachments/assets/ab81a086-de50-4d92-856e-c30f9d578226)
![trino2](https://github.com/user-attachments/assets/7a0f6c40-8662-4368-97fa-7948abe68129)

coordinator=true
node-scheduler.include-coordinator=false
discovery-server.enabled=true
http-server.http.port=8080
discovery.uri=http://localhost:8080

node.environment=trino_demo
node.id=coordinator
node.data-dir=/data/trino

worker
coordinator=false
http-server.http.port=8080
discovery.uri=http://coordinatorIP:8080

node.environment=trino_demo
node.id=worker-1
node.data-dir=/data/trino

**Comment by hashhar** [comment]
check the worker logs. Is the `http://coordinatorIP:8080` reachable from the worker container (inside the container, not just from the host).
